### PR TITLE
[Bugfix] Prevent Runtime update restarts

### PR DESCRIPTION
**Pull request by ajanikow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
 - (Feature) Configurable ArangoD Port
 - (Feature) Allow to exclude metrics
 - (Feature) Do not restart member if all pods in group are not ready
+- (Feature) Prevent Runtime update restarts
```
Comment (Robert-Stam) [inline CHANGELOG.md:40]: Feature --> Fix?

File: pkg/apis/deployment/v1/conditions_params.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const (
+	ConditionParamContainerUpdatingName ConditionParam = "updatingContainerName"
+)
+
+// ConditionParam is a strongly typed condition parameter
+type ConditionParam string
+
+type ConditionParams map[ConditionParam]string
+
+func (c ConditionParams) Equal(b ConditionParams) bool {
```
Comment (Robert-Stam) [inline pkg/apis/deployment/v1/conditions_params.go:32]: comments missing

File: pkg/apis/deployment/v1/conditions_params.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
+//
+// DISCLAIMER
+//
+// Copyright 2016-2022 ArangoDB GmbH, Cologne, Germany
```
Comment (Robert-Stam) [inline pkg/apis/deployment/v1/conditions_params.go:4]: not sure about copyright notice in operator, however in Oasis it's 
`Copyright 2022` for new files and `Copyright x-2022`
 for updates

File: pkg/apis/deployment/v1/conditions_params.go [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+package v1
+
+const (
+	ConditionParamContainerUpdatingName ConditionParam = "updatingContainerName"
```
Comment (Robert-Stam) [inline pkg/apis/deployment/v1/conditions_params.go:24]: comments missing

File: pkg/apis/deployment/v2alpha1/conditions_params.go [hunk 5]
```diff
@@ -0,0 +1,44 @@
+//
+// DISCLAIMER
+//
+// Copyright 2016-2022 ArangoDB GmbH, Cologne, Germany
```
Comment (Robert-Stam) [inline pkg/apis/deployment/v2alpha1/conditions_params.go:4]: 2022 only?

**Review by jwierzbo — APPROVED** [review]

**Pull request merged**